lib/date_time/Kconfig

```diff
@@ -25,9 +25,9 @@ config DATE_TIME_NTP
 	select SNTP
 	default y
 
-config DATE_TIME_THREAD_SIZE
+config DATE_TIME_THREAD_STACK_SIZE
 	int "Stack size of the thread maintaining date time"
-	default 1024
+	default 1152
```

Comment:
The commit with this change has been merged into master in PR#https://github.com/nrfconnect/sdk-nrf/pull/5341